problemsets-mat223/modules/module8-exercises.tex

```diff
@@ -46,6 +46,51 @@
 						F}$.
 				\end{enumerate}
 		\end{enumerate}
+		\begin{solution}
+			\begin{enumerate}
+				\item 
+					\begin{enumerate}
+						\item $[\vec{u}]_{\mathcal{E}} = \mat{1\\8}$ \hspace{4mm} $[\vec{v}]_{\mathcal{E}} = \mat{-1\\3}$ \hspace{4mm} $
+							[\vec{w}]_{\mathcal{E}} = \mat{2\\0}$
+						\item $[\vec{u}]_{\mathcal{A}} = \mat{3\\-2}$ \hspace{4mm} $[\vec{v}]_{\mathcal{A}} = \mat{1\\-1}$ \hspace{4mm} 
+							$[\vec{w}]_{\mathcal{A}} = \mat{2/11\\4/11}$
+						\item $[\vec{u}]_{\mathcal{B}} = \mat{1/2\\1}$ \hspace{5mm} $[\vec{v}]_{\mathcal{B}} = \mat{1/2\\-1{\tiny }}$ 
+							\hspace{5mm} $[\vec{w}]_{\mathcal{B}} = \mat{-5/11\\2}$
+					\end{enumerate}
+				\item
+					\begin{enumerate}
+						\item $[\vec{q}]_{\mathcal{E}} = \mat{0\\11\\-4}$ \hspace{4mm} $[\vec{r}]_{\mathcal{E}} = \mat{5\\-12 \\8}$ 
+							\hspace{4mm} $[\vec{s}]_{\mathcal{E}} = \mat{1\\0\\1}$
+						\item Given $\mathcal{D} = \left\{ \mat{1\\2\\0}_{\mathcal{E}}, \mat{-3\\5\\-4}_{\mathcal{E}}, 
+							\mat{-8\\4\\11}_{\mathcal{E}} \right\}$. Thus to find $[\vec{q}]_{\mathcal{D}}$, we need to find scalars 
+							$x, y, z$ such that $$x \mat{1\\2\\0} + y \mat{-3\\5\\-4} +z \mat{-8\\4\\11} = \mat{0\\11\\-4}$$ which 
```

Comment:
No `$$`, ever. Always use `\[...\]`. Run this through my latex beautifier and it will automatically fix that part.